lib/routes/universities/thu/career.js

```diff
@@ -1,13 +1,33 @@
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
-const resolve_url = require('url').resolve;
+const { parseDate } = require('@/utils/parse-date');
 
 const base_url = 'https://career.cic.tsinghua.edu.cn/';
+const route = 'xsglxt/f/jyxt/anony/xxfb';
 
 module.exports = async (ctx) => {
-    const page = 1;
+    const list = await fetchPage(1);
 
-    const route = 'xsglxt/f/jyxt/anony/xxfb';
+    const title = '招聘信息';
+    ctx.state.data = {
+        title: '清华大学 - ' + title,
+        link: base_url + route,
+        description: '清华大学学生职业发展指导中心 - ' + title,
+        item: await Promise.all(
+            list.map(
+                async (item) =>
+                    await ctx.cache.tryGet(item.link, async () => {
+                        const res = await got.get(item.link);
+                        const content = cheerio.load(res.data);
+                        item.description = content("div[class='sideleft left'], div[class='content teacher']").html();
```

Comment:
少用 attr 选择器，这地方如果两个同时存在那 html 只能拿到第一个，如果确认两个不会同时存在就没问题

```suggestion
                        item.description = content("div.sideleft.left, div.content.teacher").html();
```

Reply:
感谢你的建议😁